frontend/src/components/DropdownListItems.js

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import {Button, Dropdown, Menu} from "antd";
+import {CloseOutlined, LineOutlined, MenuOutlined} from "@ant-design/icons";
+
+const DropdownListItems = ({ListItems}) => {
+  const menuListItems = (
+    <Menu>
+      {ListItems ? ListItems.map((item, i) => <Menu.Item key={i.toString()}>{item}</Menu.Item>) : <Menu.Item/>}
+    </Menu>
+  );
+
+  return (
+    ListItems && ListItems.length > 1 ? 
+    <Dropdown overlay={menuListItems} placement="bottomRight">
+      <div>
+        <MenuOutlined /> {ListItems && ListItems[0]}
+      </div>
+    </Dropdown>
+    :
+    <div>
+      {ListItems ? <LineOutlined /> : <CloseOutlined />} {ListItems && ListItems[0]}
```

Comment:
Here the CloseOutlined will never show, because even if there is no sequence (sequence value is "")  the ListItems will be true. You would need to check for empty string to show the CloseOutlined.

Reply:
I did change that (in the test PR) now the sequences for the 3 prime and 5 prime index will have empty list instead of a list with an empty string.